tools/plugins/shared/webpack/webpack.config.ts

```diff
@@ -0,0 +1,215 @@
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+import ESLintPlugin from 'eslint-webpack-plugin';
+import ForkTsCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin';
+import path from 'path';
+import ReplaceInFileWebpackPlugin from 'replace-in-file-webpack-plugin';
+import { Configuration } from 'webpack';
+import LiveReloadPlugin from 'webpack-livereload-plugin';
+
+import { DIST_DIR } from './constants';
+import { getPackageJson, getPluginJson, getEntries, isWSL } from './utils';
+
+function skipFiles(f: string): boolean {
+  if (f.includes('/dist/')) {
+    // avoid copying files already in dist
+    return false;
+  }
+  if (f.includes('/tsconfig.json')) {
+    // avoid copying tsconfig.json
+    return false;
+  }
+  return true;
+}
```

Comment:
since for core plugins, there is no `src/` folder, we need to omit some files / folders that we don't need to pack.